### ISSUE TITLE
Replace 'ignored type parameters in with _

### ISSUE THREAD

**Issue by TysonMN** [elmish/Elmish.WPF#422]

**Comment by TysonMN** [comment]
Thanks for creating that PR.  When I created this, I believe that I was thinking that the `_` would be more directly visible to the user.  Later though, I realized that it would be displayed as a fresh type parameter.  Specifically, here is difference.

With `'ignored`
![2021-09-12_07-16-56_683_devenv](https://user-images.githubusercontent.com/34664007/132987321-b61dcfdd-c8e4-4def-8751-35baed617205.png)

With `_`
![2021-09-12_07-17-36_684_devenv](https://user-images.githubusercontent.com/34664007/132987333-fc0289e4-857d-4712-9bae-fcf815966609.png)

Now that I see these side-by-side, I prefer the current use of `'ignored`.  Hopefully you don't mind me closing that PR after you put in the time to create it.
